Update the documentation to describe using symlinks for sharing web identity token file for 'repository-s3' plugin (#3021)
opensearch-project/documentation-website: _tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore.md

   ```

   If you don't want to configure AWS access and secret keys, modify the following `opensearch.yml` setting. Make sure the file is accessible by the `repository-s3` plugin:
   
   ```yml
   s3.client.default.identity_token_file: /usr/share/opensearch/plugins/repository-s3/token
   ```

>    If copying is not an option, you could alternatively create a symlink to the web identity token file  inside `${OPENSEARCH_PATH_CONFIG}` folder:

Reviewer: ```suggestion
   If copying is not an option, you can create a symlink to the web identity token file in the `${OPENSEARCH_PATH_CONFIG}` folder:
```

   ```
   ln -s $AWS_WEB_IDENTITY_TOKEN_FILE "${OPENSEARCH_PATH_CONFIG}/aws-web-identity-token-file"
   ```

>    And reference it by `repository-s3` plugin using the relative path (that is going to be resolved against `${OPENSEARCH_PATH_CONFIG}`):

Reviewer: ```suggestion
   You can reference the web identity token file in the following `opensearch.yml` setting by specifying the relative path that is resolved against `${OPENSEARCH_PATH_CONFIG}`:
```
